contributors/design-proposals/propagation.md

```diff
@@ -73,13 +79,18 @@ type VolumeMount struct {
 	// Required.
 	MountPath string `json:"mountPath"`
 	// Optional.
-	Propagation string `json:"propagation"`
+	Propagation PropagationMode `json:"propagation,omitempty"`
```

Comment:
> This field name is very vague... Propagation of what? In what direction?

Renamed to `MountPropagation` with more documentation around. Still, I don't want to put whole kernel documentation there.

> So is the file name

What file name?